packages/amplify-provider-awscloudformation/lib/amplify-service-manager.js

```diff
@@ -0,0 +1,344 @@
+const fs = require('fs-extra');
+const path = require('path');
+const inquirer = require('inquirer');
+const sequential = require('promise-sequential');
+const S3 = require('../src/aws-utils/aws-s3');
+const { getConfiguredAmplifyClient } = require('../src/aws-utils/aws-amplify');
+const constants = require('./constants');
+
+async function init(amplifyServiceParams) {
+  const { context, awsConfig, projectName, envName, stackName } = amplifyServiceParams;
+
+  const amplifyClient = await getConfiguredAmplifyClient(context, awsConfig);
+
+  let amplifyAppId;
+
+  if (context.exeInfo && context.exeInfo.inputParams && context.exeInfo.inputParams.amplify && context.exeInfo.inputParams.amplify.appId) {
+    const inputAmplifyAppId = context.exeInfo.inputParams.amplify.appId;
+    try {
+      const getAppResult = await amplifyClient
+        .getApp({
+          appId: inputAmplifyAppId,
+        })
+        .promise();
+      context.print.info(`Input Amplify appId is verified: ${inputAmplifyAppId}`);
```

Comment:
Amplify AppID found: ${inputAmplifyAppId}. Amplify App name is ${getAppResult.app.name}`

Reply:
changed